.config/nextest.toml

```
@@ -12,6 +12,15 @@ retries = 1
 # Timeout tests after 5 minutes
 slow-timeout = { period = "60s", terminate-after = 5 }
 
+# isolate slow and flaky tests to run alone (no multi-thread / no other tests running)
+[test-groups]
+no-concurrency = { max-threads = 1 }
+[[profile.ci.overrides]]
+filter = 'test(generate_cursor_test) or test(test_withdraw_stake) or test(verify_latest_tx_replay_testnet_mainnet) or test(test_reconfig_with_committee_change_stress_determinism) or test(test_simulated_load_restarts) or test(test_simulated_load_rolling_restarts_all_validators) or test(test_faucet_batch) or test(test_faucet_batch_concurrent_requests) or test(test_full_node_load_migration_data) or test(test_full_node_sync_flood_determinism) or test(test_passive_reconfig_determinism) or test(test_reconfig_with_committee_change_stress_determinism) or test(test_hash_collections) or test(test_net_determinism) or test(test_epoch_data)'
```

I think this will work

```suggestion
filter = '''
test(generate_cursor_test) \
or test(test_withdraw_stake) \
or test(verify_latest_tx_replay_testnet_mainnet) \
or test(test_reconfig_with_committee_change_stress_determinism) \
or test(test_simulated_load_restarts) \
or test(test_simulated_load_rolling_restarts_all_validators) \
or test(test_faucet_batch) \
or test(test_faucet_batch_concurrent_requests) \
or test(test_full_node_load_migration_data) \
or test(test_full_node_sync_flood_determinism) \
or test(test_passive_reconfig_determinism) \
or test(test_reconfig_with_committee_change_stress_determinism) \
or test(test_hash_collections) \
or test(test_net_determinism) \
or test(test_epoch_data)'''
```